get_github_coderabbit_review/tests/integration/test_github_integration.py

```
@@ -0,0 +1,466 @@
+        """Test PR comments fetching with network error."""
+        mock_run.return_value = MagicMock(
+            returncode=1,
+            stdout="",
+            stderr=MOCK_GH_ERROR_RESPONSES["network_error"]["stderr"]
+        )
+        
+        with self.assertRaises(NetworkError):
+            self.client.fetch_pr_comments(self.sample_pr_url)
+    
+    @patch('subprocess.run')
+    def test_parse_pr_url_valid(self, mock_run):
```

Ruff (0.13.1)

Unused method argument: `mock_run`

(ARG002)